src/Buttons.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Buttons({ buttons, onClick, selectId }) {
+  return (
+    <ul>
+      {buttons.map((button) => (
+        <li key={button.id}>
+          <button type="button" onClick={() => onClick(button.id)}>
+            {
+              `${button.name}${
+                selectId === button.id ? '(V)' : ''
+              }`
+            }
```

Comment:
```suggestion
            {button.name}
            {selectId === button.id ? '(V)' : null}
```

개인적으로 가독성에 좋지 않은 것 같아요. 이런 방식으로 처리할 순 없을까요?